wordpress.org/public_html/wp-content/plugins/wporg-gp-custom-errors/wporg-gp-custom-errors.php

```diff
@@ -169,10 +186,10 @@ public function error_timezone_date_format( $original, $translation, $gp_origina
 		if ( ! $this->is_core_project( $gp_original ) ) {
 			return true;
 		}
-		if ( is_null( $gp_original->context ) ) {
+		if ( is_null( $gp_original->comment ) ) {
 			return true;
 		}
-		if ( ! str_contains( $gp_original->context, 'timezone date format' ) ) {
+		if ( strpos( $gp_original->comment, 'https://www.php.net/manual/datetime.format.php' ) === false ) {
```

Comment:
This can keep using `str_contains()` as WordPress has a polyfill for it. Unless you want to do a case-insensitive check then this should be a `stripos`.